cloud/scope/vpc_cluster.go

```diff
@@ -686,3 +742,326 @@ func (s *VPCClusterScope) buildCOSObjectHRef() (*string, error) {
 	s.V(3).Info("building image ref", "href", href)
 	return ptr.To(href), nil
 }
+
+// ReconcileSubnets reconciles the VPC Subnet(s).
+// For Subnets, we collect all of the required subnets, for each Plane, and reconcile them individually. Requeing if one is missing or just created. Reconciliation is attempted on all subnets each loop, to prevent single subnet creation per reconciliation loop.
+func (s *VPCClusterScope) ReconcileSubnets() (bool, error) {
+	var subnets []infrav1beta2.Subnet
+	var err error
```

Comment:
I believe I need this declared here, so that when assigning `subnets` below, it is not redefined. The linting at least requires the declarations first, before assigning.